api/src/main/java/io/grpc/CallOptions.java

```diff
@@ -203,6 +207,46 @@ public CallOptions withoutWaitForReady() {
     return builder.build();
   }
 
+  /**
+   * Specifies how many bytes must be queued before the call will leave the
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md">
```

Comment:
Wait-for-ready is completely different; it is unrelated to this. Maybe: "before the call is considered not ready to send more messages."